crates/cxx-qt-gen/src/syntax/qtfile.rs

```diff
@@ -58,3 +58,91 @@ pub fn parse_qt_file(path: impl AsRef<std::path::Path>) -> Result<CxxQtFile> {
         syn::parse_str(&source)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::tests::assert_tokens_eq;
+    use crate::CxxQtItem::CxxQt;
+    use quote::quote;
+    use std::env;
+    use std::path::PathBuf;
+    use syn::parse_quote;
+
+    #[test]
+    fn test_parse_qt_file() {
+        let manifest_dir = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap());
+
+        let qt_file = parse_qt_file(manifest_dir.join("test_inputs/properties.rs")).unwrap();
+
+        assert!(qt_file.attrs.is_empty());
+        assert_eq!(qt_file.items.len(), 1);
+        assert!(matches!(qt_file.items.first(), Some(CxxQt(_))))
+    }
+
+    #[test]
+    fn test_parse_qt_file_shebang_strip() {
+        let manifest_dir = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap());
+
+        let qt_file = parse_qt_file(manifest_dir.join("test_inputs/shebang.rs")).unwrap();
+
+        assert_eq!(qt_file.items.len(), 3);
+        // Testing the debug formatter
+        for item in qt_file.items {
+            println!("[Debug] item: {:?}", item);
+        }
```

Comment:
this should be a test in `qtitem.rs` i think? And there you could do things much tighter so you could have

```rust
let cxx: CxxQtItem = parse_quote!{
  #[cxx::bridge]
  mod ffi {}
};
let debug_formatted = format!("{:?}", cxx);
assert_eq!(debug_formatted, ...);
```
and you can then do the same for Item / Cxx / CxxQt